PreFreeSurfer/PreFreeSurferPipeline.sh

```diff
@@ -342,6 +360,10 @@ GradientDistortionCoeffs=`opts_GetOpt1 "--gdcoeffs" $@`
 AvgrdcSTRING=`opts_GetOpt1 "--avgrdcmethod" $@`
 TopupConfig=`opts_GetOpt1 "--topupconfig" $@`
 BiasFieldSmoothingSigma=`opts_GetOpt1 "--bfsigma" $@`
+CustomBrain=`opts_GetOpt1 "--custombrain" $@`
+
+# Defaults
+CustomBrain=`opts_DefaultOpt $CustomBrain NONE`
 
 #NOTE: currently is only used in gradient distortion correction of spin echo fieldmaps to topup
 #not currently in usage, either, because of this very limited use
```

Comment:
@coalsont   Do you know why this comment is here?  It certainly seems as if the `UseJacobian` option is "in use" (in the call to T2wToT1wDistortionCorrectAndReg.sh)

Reply:
Not sure what the history of these comments is, but the distinction is that we don't use the jacobian of an atlas registration, we take the jacobian of only the "signal pileup" warpfield during a distortion correction, and resample it to the necessary space before applying it to the values.

Reply:
The `#not currently in usage ...` comment makes no sense to me, and in fact contradicts itself.
Let's change the two lines to the following:
```
#NOTE: currently UseJacobian only affects whether the spin echo field maps 
# get modulated by the gradient distortion correction warpfield 
# (T2wToT1wDistortionCorrectAndReg -> TopupPreprocessingAll)
```

Reply:
Does it also tell topup to use the jacobian when registering the fieldmap?

I would remove the word "currently" unless we think that it should be doing additional things.

Reply:
Looks like the topup script applies the fieldmap jacobian to the output regardless.  So, I think this option doesn't need to do anything else in the future, so "currently" can be dropped.  It might be arguable that it should always be in effect, and thus not an option, but that isn't within the scope of this PR.

Reply:
We don't jacobian modulate the T1w and T2w images because that wouldn't do much.  I assume that is what I was getting at.  Also modulating the SE fieldmaps with the Jacobian wouldn't do much either.

Reply:
Sure, drop "currently" from my proposed rewording.  It is true that the topup script applies the fieldmap jacobian "regardless" to the SEFMs and Scout, but as far as I can tell, that is only to generate `*_dc_jac` output files in ${WD}, and it doesn't appear that those actually get used in anything downstream (e.g., they don't get renamed/copied to anything else within TopupPreprocessingAll or T2wToT1wDistortionCorrectAndReg).  (Which isn't to say that the Jacobian.nii.gz itself doesn't get used somewhere downstream of PreFS -- really hard to trace this through).

Reply:
It doesn't

Reply:
"currently" is taken out.